frontend/src/test/java/com/javabeast/service/liveLandingPageNavigation.java

```diff
@@ -0,0 +1,279 @@
+package com.javabeast.service;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+// navigation link tests. verification that buttons are linking to correct URL.
+public class liveLandingPageNavigation {
+
+    private WebDriver driver = new ChromeDriver();
+
+    // faq navigation - top nav bar
+    @Test
+    public void callFAQNav() throws InterruptedException {
+        driver.get("https://bike-tracker.com/"
+        );
+        Thread.sleep(1000);
+        driver.findElement(By.linkText("FAQ")).click();
+        String currentUrl = driver.getCurrentUrl();
+        Assert.assertEquals(currentUrl, "https://bike-tracker.com/faq");
+        driver.quit();
+    }
+
+    // features navigation - top nav bar
+    @Test
+    public void callFeaturesNav() throws InterruptedException {
+        driver.get("https://bike-tracker.com/"
```

Comment:
The URL is hardcoded to the live website, if you make this a 

private static final String URL = " url goes here";

Then we can switch it between localhost, live and staging